account-kit/react/src/components/auth/card/loading/oauth.tsx

```diff
@@ -10,14 +10,17 @@ export const CompletingOAuth = () => {
   const { isConnected } = useSignerStatus();
   const { setAuthStep, authStep } = useAuthContext("oauth_completing");
   const { authenticate } = useOAuthVerify({ config: authStep.config });
+  const oauthWasCancelled = authStep.error?.name === "OauthCancelledError";
```

Comment:
Dumb question is this the same across all Oauth providers? 

Is this an error that we propagate down? Should we store it in a const file somewhere?

Reply:
Actually, you're right, let me use a constant here. It's the name of a class in the SDK.

Reply:
would the error be `instanceof OauthCancelledError` here?

Reply:
Yes it would, but `OauthCancelledError` is exported from `@account-kit/signer`, but not from any of the direct dependencies `@account-kit/react` (core, infra, and logging). So I need to either expose it from `core` or add `signer` as a dependency to `react`.

Exposing it from `core` makes a lot of sense. One question is whether it should be exposed from `react` as well.

Reply:
react should have implicit access to `@account-kit/signer` because of its dependency on core right? I think we should just add signer as a dependency to `react` potentially if we don't want it to be implicit. because of the monorepo setup and the fact that we keep versions tied together, it should be fine because core `vX` will always depend on the same signer `vX` as react